### DISCUSSION TITLE
immich upload

### DISCUSSION THREAD

**Discussion by RaymondDay61** [immich-app/immich#11448] [Community Guides]
Wow been working on this a long time then found out were to get the key you got to click on the round icon of your name for settings and make a key.

Then on the command line I got long in.

Want to copy all my Windows photos and they are in years and month folders.

Got this on a Ubuntu server and mounted that Windows drive.

After got log in I tested uploading the images. But just got "No files found, exiting"

Here is what commands I did not get errors:

root@8652501ebe63:/usr/src/app# immich upload --dry-run -r /media/Data/Pictures/
Crawling for assets...
No files found, exiting
root@8652501ebe63:/usr/src/app# immich upload --dry-run -r /media/Data/Pictures/*
Crawling for assets...
No files found, exiting
root@8652501ebe63:/usr/src/app# immich upload --dry-run -r /media/Data/Pictures
Crawling for assets...
No files found, exiting
root@8652501ebe63:/usr/src/app# immich upload --dry-run -r /media/Data/Pictures/
Crawling for assets...
No files found, exiting
root@8652501ebe63:/usr/src/app# immich upload --dry-run --recursive /media/Data/Pictures/
Crawling for assets...
No files found, exiting
root@8652501ebe63:/usr/src/app# immich upload --dry-run --recursive "/media/Data/Pictures/2024/(01) Jan"
Crawling for assets...
No files found, exiting
root@8652501ebe63:/usr/src/app# immich upload --dry-run --recursive "/media/Data/Pictures/2024/(01) Jan*"
Crawling for assets...
No files found, exiting
root@8652501ebe63:/usr/src/app# immich upload --dry-run --recursive "/media/Data/Pictures/2024/*
> ^C
root@8652501ebe63:/usr/src/app# immich upload --dry-run --recursive /media/Data/Pictures/2024/*

What would be the right command?

Thank you.

**Comment by alextran1502** [comment]
Can you check what CLI version you are using? `immich --version`. And where were you running the command from?

**Reply by RaymondDay61** [reply]
I guess because it's in a docker can't see my /mdeia/Data/Pictures:

root@8652501ebe63:/usr/src/app# immich --version
2.2.10
root@8652501ebe63:/usr/src/app# pwd
/usr/src/app
root@8652501ebe63:/usr/src/app# ls /
LICENSE  boot   dev  home  lib64     media  opt   root  sbin  sys  usr
bin      build  etc  lib   licenses  mnt    proc  run   srv   tmp  var
root@8652501ebe63:/usr/src/app# ls /media
root@8652501ebe63:/usr/src/app# cd /media
root@8652501ebe63:/media# ls
root@8652501ebe63:/media#

How can I fix this I mounted it like this: mount -t ntfs-3g /dev/sdg2 /media/Data/

It there a way to mount it right in docker this is the 1st time using docker for me just installed it for immich.

Thank you.

**Reply by RaymondDay61** [reply]
I am long in the docker part and when I go to list drives is don't have a fdisk command:

root@8652501ebe63:/# fdisk -l
bash: fdisk: command not found
root@8652501ebe63:/#

Now I know why some don't like docker.

**Reply by alextran1502** [reply]
Usually you would check for those things on the host. Docker is meant to be built with the necessary dependencies for the application that you are deploying

**Comment by RaymondDay61** [comment]
Yes and I thought I could just copy all my Windows photos to it easy I was staring to and did for about 2 hours and got about 20% of them.

Because they are in folders.

So copied them to a SSD and mounted it on my Ubuntu server.  That's before I know docker can't see it.

There and no recursive on the URL of Immich to upload them?

If not maybe in the next version. Wasted a lot of time trying to get my photos on my Windows 11 to it.

**Reply by alextran1502** [reply]
You can install immich CLI on Windows 11 and then upload it to your instance, right? Also you can try immich-go if you don't want to install Nodejs.

**Comment by RaymondDay61** [comment]
I did install immich-go but I can't get it to work. I gave up. I don't want to install it on my Windows 11. Well I would install it if it's a little one that would auto update my photos. Do they have that?
